### ISSUE TITLE
AVT 1 - Pagination: DAP violations 

### ISSUE THREAD

**Issue by snidersd** [IBM/carbon-components-react#2236]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
Run DAP on the pagination component
See screenshot below for violations:
<img width="1104" alt="Screen Shot 2019-04-18 at 11 06 57 AM" src="https://user-images.githubusercontent.com/21676914/56375070-30b58000-61ca-11e9-82e0-4f674a3d3de0.png">

Note: The label ID's are not unique as displayed in the screenshot.
